misk-redis-testing/src/test/kotlin/misk/redis/AbstractRedisTest.kt

```diff
@@ -1,37 +1,23 @@
 package misk.redis
 
 import com.google.inject.Module
-import com.google.inject.util.Modules
-import misk.testing.MiskTest
-import misk.testing.MiskTestModule
-import misk.time.FakeClockModule
 import okio.ByteString.Companion.encodeUtf8
+import org.assertj.core.api.Assertions
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Assertions.assertThatThrownBy
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertThrows
 import redis.clients.jedis.args.ListDirection
-import wisp.time.FakeClock
-import java.time.Duration
-import javax.inject.Inject
-import kotlin.random.Random
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
-import kotlin.test.assertNotNull
 import kotlin.test.assertNull
 import kotlin.test.assertTrue
 
-@MiskTest
-class FakeRedisTest {
+abstract class AbstractRedisTest {
   @Suppress("unused")
-  @MiskTestModule
-  val module: Module = Modules.combine(
-    FakeClockModule(),
-    RedisTestModule(Random(1977)), // Hardcoded random seed for hrandfield* test determinism.
-  )
+  abstract val module: Module
```

Comment:
Is this field used or necessary given the abstract class doesn't have `@MiskTest` or `@MiskTestModule` annotations? I imagine that setting the module on the implementation classes is sufficiently without the abstract placeholder.